docs/releases/upgrading/upgrade-v10.0.md

```diff
@@ -0,0 +1,110 @@
+# Upgrading from v9.0 to v10.0
+
+## Prerequisites
+
+The steps listed in this article require an existing local installation of InvenioRDM v9.0, please make sure that this is given!
+
+If unsure, run `invenio-cli install` from inside the instance directory before executing the listed steps.
+
+!!! warning "Backup"
+
+    Always backup your database and files before you try to perform an upgrade.
+
+!!! info "Older versions"
+
+    In case you have an InvenioRDM installation older than v9.0, you can gradually upgrade using the existing instructions to v9.0 and afterwards continue from here.
```

Comment:
```suggestion
    In case you have an InvenioRDM installation older than v9, you can gradually upgrade using the existing instructions to v9 and afterwards continue from here.
```